docs/auto-instrumentation-install.md

```diff
@@ -533,10 +529,93 @@ kubectl get certificates
 # splunk-otel-collector-operator-serving-cert   True    splunk-otel-collector-operator-controller-manager-service-cert   5m
 ```
 
-##### Using a Self-Signed Certificate for the Webhook
+#### TLS Certificate Requirement for Kubernetes Operator Webhooks
+
+In Kubernetes, the API server communicates with operator webhook components over HTTPS, which requires a valid TLS certificate that the API server trusts. The operator supports several methods for configuring the required certificate, each with different levels of complexity and security.
+
+---
+
+##### 1. **Using a Self-Signed Certificate Generated by the Chart**
+
+This is the default and simplest method for generating a TLS certificate. It automatically creates a self-signed certificate for the webhook. It is suitable for internal environments or testing purposes but may not be trusted by clients outside your cluster.
+
+**Configuration:**
+- Set `admissionWebhooks.certManager.enabled` to `false` and `admissionWebhooks.autoGenerateCert.enabled` to `true`.
+- Helm generates a self-signed certificate, valid for 10 years, and creates a secret for the webhook.
+- The certificate is automatically recreated on every Helm upgrade.
```

Comment:
```suggestion
- The certificate is automatically recreated on every Helm upgrade. This behavior can be disabled by setting the config `operator.admissionWebhooks.autoGenerateCert.recreate` to `false`
```

Reply:
I found a [sizable issue](https://github.com/open-telemetry/opentelemetry-helm-charts/blob/be4c2659ec3005b1dbce238df7d730c5461746ba/charts/opentelemetry-operator/templates/admission-webhooks/operator-webhook.yaml#L12) related to this I'm trying to test fixes for locally and make a fix for upstream.

Reply:
Can you clarify what is the issue? secret being recreated on each upgrade?

Reply:
https://github.com/open-telemetry/opentelemetry-helm-charts/pull/1531

Reply:
Added this change.